widgets/gauge/options.go

```diff
@@ -201,3 +205,17 @@ func BorderTitleAlign(h align.Horizontal) Option {
 		opts.borderTitleHAlign = h
 	})
 }
+
+// Threshold configures the Gauge to display a vertical threshold line at value
+// t. If the progress is set by a call to Percent(), t represents a percentage,
+// e.g. "40" means line is displayed at 40%. If the progress is set by a call to
```

Comment:
One of the principles I was trying to follow in Termdash is that invalid configuration will notify the user rather than silently take and ignore the value. 

How would you feel about modifying this, so that it returns an error if the user sets the threshold to less than 0? We could reuse the existing `validate` function.